README.Rmd

```diff
@@ -151,7 +151,8 @@ The (R)md to XML to (R)md loop on which `tinkr` is based is slightly lossy becau
 
  * lists can be created with either "+", "-" or "*". When using `tinkr`, the (R)md after editing will only use "-" for lists. 
  
- * Links built like `[word][smallref]` and bottom `[smallref]: URL` become `[word](URL)`.
+ * Links built like `[word][smallref]` with a bottom anchor `[smallref]: URL`
+  will have the anchor move to the bottom of the document.
```

Comment:
```suggestion
  will have the anchor moved to the bottom of the document.
```

It seems fitting that anchors sink. :anchor:

Reply:
I thought so, too! [The documentation for reference-style links](https://www.markdownguide.org/basic-syntax/#reference-style-links) calls this the "second part" of the link, which I felt was kind of clunky; anchor seemed appropriate. 

And thank you for fixing the grammar! I was going to say that your correction was not right, but then I realized that your edit puts it correctly in future perfect tense and I was trying to do future imperfect.

Reply:
Ah you'd know better about grammar than I! I might be wrong!

Reply:
Au contraire! I speak English natively, which means that you likely know a whole lot more about English grammar than I do! I had to look up what tense I was using.